JetAnalyzers/bin/JERWriter.h

```diff
@@ -22,8 +22,10 @@
 #include <vector>
 #include <map>
 #include <set>
+#include <math.h>
 
-
+#include "TFile.h"
+#include "TProfile.h"
```

Comment:
Move lines 27-28 right below the TF1.h library and change the TF1.h library from having <> around it to "".